lib/quantum/execution_broadcaster.ex

```diff
@@ -143,7 +141,7 @@ defmodule Quantum.ExecutionBroadcaster do
       ) do
     debug_logging &&
       Logger.debug(fn ->
-        "[#{inspect(Node.self())}][#{__MODULE__}] Removing job #{inspect(name)}"
+        {"Removing job", node: Node.self(), name: name}
```

Comment:
Would it make sense to add the job itself as a metadata entry instead of just the name? (I haven't really used the new logger capabilities yet, I don't know if this is a good or bad idea...)

Reply:
Is the Job struct JSON-serializable? That is how I generally use Logger metadata.

Reply:
Not at the moment.

How could the metadata be used other then JSON loggers?

Reply:
The standard metadata format. I think most other use cases generally include JSON.

Reply:
Hm, then I think we should leave it as is.